src/components/DateRangePicker.res

```diff
@@ -540,7 +540,9 @@ module Base = {
     }
 
     let buttonText = {
-      showTime
+      startDateVal->Js.String2.length === 0 && endDateVal->Js.String2.length === 0
```

Comment:
can we start using the rescript core changes from every PR?
For Reference - [Conversions](https://github.com/rescript-association/rescript-core/blob/main/migration/migration.toml)